### PR TITLE
Revert coverage reports

### PR DESCRIPTION
**Pull request by buckle90**
Talked to Ken and decided to revert the coverage report changes and cherry pick the timeout fix I made in order to unblock us from bumping the version. I also updated a test to check that the timeout is being set correctly

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,8 @@
-language: android
```
Comment (KennethNickles) [inline .travis.yml:1]: I think you need to leave all the version updates and travis changes for this to pass on travisci.

File: torque-core/src/test/kotlin/com/workday/torque/ScreenRecorderSpec.kt [hunk 2]
```diff
@@ -47,7 +48,18 @@ class ScreenRecorderSpec : Spek(
                     it[0] == "-s" && it[1] == adbDevice.id && it[2] == "shell" &&
                             it[3] == "screenrecord someDeviceDir/videos/id/someTestClass/someTestMethod/test_recording.mp4 --time-limit $DEFAULT_PER_CHUNK_TIMEOUT_SECONDS --size 720x1440"
                 }
-                processRunner.runAdb(recordCommandAndArgsMatcher, any(), any(), any(), any(), any(), any())
+                val expectedTimeout = Timeout(
+                        DEFAULT_PER_CHUNK_TIMEOUT_SECONDS.toInt() + TIMEOUT_BUFFER_SECONDS,
+                        TimeUnit.SECONDS)
+
+                processRunner.runAdb(commandAndArgs = recordCommandAndArgsMatcher,
+                        timeout = expectedTimeout,
```
Comment (KennethNickles) [inline torque-core/src/test/kotlin/com/workday/torque/ScreenRecorderSpec.kt:56]: looks good

File: torque-core/src/test/kotlin/com/workday/torque/TestChunkRunnerSpec.kt [hunk 3]
```diff
@@ -85,7 +85,7 @@ class TestChunkRunnerSpec : Spek(
 
                     val adbDeviceTestResults = runBlocking {
                         testChunkRunner.run(args, testChunk)
-                    }
+                    }!!
```
Comment (KennethNickles) [inline torque-core/src/test/kotlin/com/workday/torque/TestChunkRunnerSpec.kt:88]: You can probably remove this.

File: torque-core/src/main/kotlin/com/workday/torque/ScreenRecorder.kt [hunk 4]
```diff
@@ -42,7 +42,8 @@ class ScreenRecorder(private val adbDevice: AdbDevice,
         ),
                 destroyOnUnsubscribe = true,
                 timeout = Timeout(timeoutSeconds.toInt() + TIMEOUT_BUFFER_SECONDS, TimeUnit.SECONDS))
-                .ofType(Notification.Exit::class.java)
+                .ofType(
```
Comment (KennethNickles) [inline torque-core/src/main/kotlin/com/workday/torque/ScreenRecorder.kt:45]: formatting

File: gradle/wrapper/gradle-wrapper.properties [hunk 5]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10-all.zip
```
Comment (KennethNickles) [inline gradle/wrapper/gradle-wrapper.properties:6]: keep this

**Review by KennethNickles — CHANGES_REQUESTED** [review]

**Comment by hujim** [comment]
> Talked to Ken and decided to revert the coverage report changes and cherry pick the timeout fix I made in order to unblock us from bumping the version. I also updated a test to check that the timeout is being set correctly

Why do you need to revert to unblock, can't you just don't supply the parameter?

**Pull request closed without merging**